src/InputManager.cpp

```diff
@@ -21,27 +21,48 @@ void InputManager::CheckInput() {
 void InputManager::MouseInput() {
     double xpos, ypos;
 
-    if ( shiftKey && glfwGetMouseButton( window, GLFW_MOUSE_BUTTON_LEFT ) == GLFW_PRESS ) {
-        glfwGetCursorPos( window, &xpos, &ypos );
-        float deltaX = ( xpos - cursorX ) * MOUSE_SENSITIVITY;
-        float deltaY = ( ypos - cursorY ) * MOUSE_SENSITIVITY;
-
-        if ( !( deltaX || deltaY ) ) {
-            return;
+    if ( sketching ) {
+        static int oldState = GLFW_RELEASE;
+        int newState = glfwGetMouseButton(window, GLFW_MOUSE_BUTTON_LEFT);
+
+        if ( newState == GLFW_PRESS ) {
+            glfwGetCursorPos( window, &xpos, &ypos );
+            float MVPX = 2.0f * ( xpos / 512.0f ) - 1;
+            float MVPY = -2.0f * ( ypos / 512.0f ) + 1;
+            if ( xpos != cursorX || ypos != cursorY ) {
+                Geometry* g = scene->GetGeometry( 1 );
```

Comment:
Uhh this will break when we instantiate / load more than one geometry into the scene. 

Maybe just hard code in a Geometry* sketch(es) in Scene and just change that instead